tests/utils/test_gzipgen.py

```diff
@@ -0,0 +1,9 @@
+from launchable.utils.gzipgen import compress
+import gzip
+
+def test_compress():
+    """Basic sanity test of """
+    encoded=b''.join(compress([b'Hello',b' ',b'world']))
+    msg = gzip.decompress(encoded)
+    print(msg)
+    assert msg==b'Hello world'
```

Comment:
Please use `eq_` from `from nose.tools import eq_` instead of `assert`.

Reply:
So the difference between `eq_` and `assert` is that the latter might get optimized away with some compilation option?